server/repository/assignments.go

```diff
@@ -24,6 +24,8 @@ const (
 	selectAssignmentsWhereExpPairSQL = `SELECT * FROM assignments WHERE experiment_id=$1 AND pair_id=$2`
 	updateAssignmentsSQL             = `UPDATE assignments SET answer='%v', duration=%v WHERE id=%v`
 	countPendingIDsSQL               = `SELECT count(id) FROM file_pairs WHERE experiment_id=$1 AND id NOT IN (SELECT pair_id FROM assignments WHERE experiment_id=$1 AND user_id=$2)`
+	countUserAssigmentsSQL           = `SELECT COUNT(*) FROM assignments WHERE experiment_id=$1 AND user_id=$1`
+	countCompleteUserAssigmentsSQL   = `SELECT COUNT(*) FROM assignments WHERE experiment_id=$1 AND user_id=$1 AND answer IS NOT null`
```

Comment:
These two queries are using $1 for both arguments, user_id should be $2

Reply:
thank you so much! fixed